docs/source/_static/install_depthai.sh

```diff
@@ -127,9 +127,6 @@ if [[ $(uname -s) == "Darwin" ]]; then
   echo "Installing global dependencies."
   bash -c "$(curl -fL https://docs.luxonis.com/install_dependencies.sh)"
 
-  echo "Upgrading brew."
-  brew update
-
```

Comment:
@PetrNovota this is not needed anymore or was it never needed?

Reply:
this was never needed. I think this is actually relatively dangerous to do without the users permission.